packages/react-dialog/src/components/Dialog/Dialog.types.ts

```diff
@@ -2,10 +2,21 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 
 export type DialogSlots = {
   root: Slot<'div'>;
+  overlay?: Slot<'div'>;
 };
 
 type DialogCommons = {
-  // TODO Add things shared between props and state here
+  /**
+   * Whether the dialog is open or closed.
+   * @defaultvalue undefined
+   */
+  open: boolean;
```

Comment:
let's also introduce onOpenChange, similar to Tooltip's `visible` and its change callback. This will allow usage of full controlled pattern.